components/editor/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "scripts": {
-    "prepare": "lezer-generator command-k/command.grammar -o command-k/parser.js"
+    "prepare": "lezer-generator command-k/command.grammar -o command-k/parser.js",
```

Comment:
Dont' add a new command, make it part of prepare, use && to separate both commands.